### ISSUE TITLE
Avalon 6.2 is not compatible with latest mediainfo version

### ISSUE THREAD

**Issue by mohideen** [avalonmediasystem/avalon#2520]
At the University of Maryland, we are running an Avalon 6.2 version on a RHEL 7 VM. During the initial setup, we installed mediainfo version `0.7.61-1`, but one of the recent system updates updated the package to the most recent version `17.10`. This was causing an error that prevents masterfiles from being loaded to the media_objects.

```
E, [2017-12-05T21:58:02.787770 #18576] ERROR -- : Problem saving MasterFile(): Workflow name field is required. and Workflow name is not included in the list
W, [2017-12-05T21:58:08.222781 #18576]  WARN -- : Could not find workflow for: #<MasterFile:0x00000030b7a5e0>
```
The MasterFile class was not able to determine the workflow as it was not able to identify the file format. Once I downgraded mediainfo package back to `0.7.61-1`, Avalon is able to load the masterfiles correctly.

**Comment by cjcolvar** [comment]
@davidschober @joncameron Can we fit this into 6.3?  I'm guessing it wouldn't be too bad to fix and would be sized around a 3.  If we don't think we can do it, we could add a known issue with the workaround and fix it 6.3.1.

**Comment by davidschober** [comment]
Let’s put at the bottom. We have a hard deadline

**Comment by cjcolvar** [comment]
Verified by @carrickr.

**Comment by carrickr** [comment]
Verified and fixed on the avalon fork of media-info.  

@mohideen if you want a fix for 17.10 you can look into manually applying  this to your local Avalon:

https://github.com/nulib/avalon/commit/1ffb59432dbb2d8c58d0073b564c89d7a77a6ccb

@davidschober I will put in a PR to bring this fix to 6.3, but we also need to rearchitect this section of the code so that in the future when MediaInfo changes its output format, Avalon does something more than pass Fedora nils.
